app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TweetsController < ApplicationController
+  #index
+  get '/tweets' do
+    if Helper.logged_in?(session)
+      @tweets= Tweet.all
+      @user = Helper.current_user(session)
+      erb :'tweets/tweets'
+    else
+      redirect to "/login"
+     end
+  end
+
+  #new
+  get '/tweets/new' do
+    if Helper.logged_in?(session)
+      erb :'tweets/new'
+    else
+      redirect "/login"
+    end
+  end
+
+  #create
+  post '/tweets' do
+    user = Helper.current_user(session)
+    if !params[:content].empty?
+      @tweet= Tweet.create(content: params[:content])
+      @tweet.user = user
+      @tweet.save
+    else
+      redirect "/tweets/new"
+    end
+    redirect("/tweets/#{@tweet.id}")
+  end
+
+  #show
+  get '/tweets/:id' do
+    if Helper.logged_in?(session)
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'tweets/show'
+    else
+      redirect "/login"
+    end
+  end
+
+  #edit
+  get '/tweets/:id/edit' do
+    @tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?